python/lsst/cp/pipe/defects.py

```diff
@@ -716,3 +762,154 @@ def run(self, inputDefects, camera):
         return pipeBase.Struct(
             mergedDefects=merged,
         )
+
+# Subclass the MergeDefects task to reduce the input dimensions
+# from ("instrument", "detector", "exposure") to
+# ("instrument", "detector").
+
+
+class MergeDefectsCombinedConnections(MergeDefectsConnections,
+                                      dimensions=("instrument", "detector")):
+    inputDefects = cT.Input(
+        name="singleExpDefects",
+        doc="Measured defect lists.",
+        storageClass="Defects",
+        dimensions=("instrument", "detector",),
+        multiple=False,
+    )
+    secondaryInputDefects = cT.Input(
+        name="AdditionalSingleExpDefects",
```

Comment:
This default name should be `cpPartialDefectsCombinedWithFilter`.  Or maybe `cpPartialDefectsFromFlatCombined` and that implies the filter.  Furthermore, I don't know how this works ... are we only doing one flat combination in here, or does this need to support `physical_filter` dimension and `multiple=True`?

Reply:
I changed the name to `cpPartialDefectsFromFlatCombined`.

Reply:
This was failing before and @czwa suggested the solution "with Filter".